contents/blog/seed-grow-scale-devrel.mdx

```diff
@@ -0,0 +1,217 @@
+
+### Work with customers, partners, and community
+
+When scaling, it's essential to think beyond simply adding headcount. That, in itself, isn't scaling. You're only really scaling a team if further investment increases output more than the increase in input. For example, if a Developer Education team of five people produces five pieces of content a week (one piece per person), adding another person should result in the team producing ten pieces of content a week.
+
+So, how do you scale DevRel? As discussed, a specialized, focused, and efficient team is a step in the right direction. But, the real opportunity to scale is through collaboration with customers, partners, and your broader community. Let's use the word "community" to encapsulate customers, partners, and the wider community to simplify things.
+
+Firstly, you should work for and with your community to create a [flywheel effect](https://www.jimcollins.com/concepts/the-flywheel.html). You should do this in combination with programs that support and enable collaboration. Here are some examples:
+
+- **Open-source**: Open-source as much as you can to enable your community to provide feedback and contribute. At a minimum, this should include your documentation, sample code, SDKs, and API specifications. If possible, open-source your whole platform (see [Open-core](https://en.wikipedia.org/wiki/Open-core_model)).
+- **Value-add "write for us" program**: "Write for us" programs shouldn't be a cheap way to get content. But, they can be effective if they provide genuine value to the author. Beyond payment or a donation to a charity chosen by the author, this means providing detailed feedback on content to help the author improve as a writer and software developer, attributing the content to the author and promoting it with acknowledgement to help them grow their profile.
+- **Rewards & heroes program**: Have a clear reward structure for contributions, celebrate your MVPs, and when you feel there's a large enough community to justify it, create a hero or ambassador program.
+- **Meetups program**: There are two approaches you can take here:
+  1. If you have a symbiotic alignment with an existing technology trend, it's worth reaching out to existing meetup organizers and offering sponsorship. You will initially get out what you put in but, over time, this will create the desired flywheel effect.
+  2. If you're fortunate enough to have a large or highly dedicated community, it's worth considering creating a mechanism that allows your heroes or ambassadors to organize meetups. Create a meetup organizer package and help your hero set up, run, and support the meetup.
+
+As with many things in DevRel, these are unlikely to be quick wins. But, as the community grows, engages, and contributes, the velocity of the flywheel increases. As this happens, the community is increasingly responsible for the input and output as you work to scale your DevRel program.
```

Comment:
Consider adding a line about "nor should you aim to spin up all of these at the same time" too much focus on spinning up everything leads to less good overall, pick the quick wins or highest value first and get that right, then add the next.

Reply:
Added in 0bcaa31ac9573a13b06e1979f830cd8ddd9f7663